.travis.yml

```diff
@@ -7,3 +7,9 @@ services:
 script:
   - docker build -t riotdocker .
   - docker image ls riotdocker:latest
+  - git clone --depth 1 https://github.com/RIOT-OS/RIOT -b 2019.01
+  - sed -i '35i\ \ BOARDS \\' RIOT/makefiles/docker.inc.mk
```

Comment:
What is this?

Reply:
added a hint. without the fix, BOARDS won't propagated into the contianer, making this test build *everything*. We know how travis likes our full builds...

Reply:
BTW. The first two steps would have been better placed in [`before_script:`](https://docs.travis-ci.com/user/customizing-the-build), but oh well...